e2e/testdata/fn-eval/missing-fn-image/.expected/config.yaml

```diff
@@ -17,4 +17,4 @@ exitCode: 1
 image: gcr.io/kpt-fn/dne # non-existing image
 args:
   namespace: staging
-stdErr: 'Function image "gcr.io/kpt-fn/dne" doesn''t exist'
+stdErr: 'gcr.io/kpt-fn/dne:latest not found'
```

Comment:
Why we have this change? I didn't find any changes related to this.

Reply:
Earlier we were calling `docker pull` explicitly and capturing the not found error (look at the deleted code), so now that's gone, we relay the message from `docker run` as is. That error is little verbose because it's coming from docker, but I didn't find it much value in wrapping it up because the verbose message from docker is pretty clear.